plotting/private/bg_rgba2rgb.m

```diff
@@ -0,0 +1,116 @@
+function [rgb] = bg_rgba2rgb(rgba, bg, varargin)
+
+if size(bg,1) ~= size(rgba,1)
+    error('the background and data should have the same number of points in the rows');
```

Comment:
points is (or will be) ambiguous. In the future it can be either vertices or voxels. I suggest something like

"the background and color data should have the same number of rows"